stories/kubernetes_azure_suspicios_provision.yml

```diff
@@ -0,0 +1,26 @@
+category:
+  - Cloud Security
+channel: ESCU
+creation_date: '2020-03-24'
+description: This story addresses investigation against Kubernetes operations by providing information on items such as source ip, user agent, cluster names.
+detections:
+  - detection_id: 898c0f86-1f3e-49c8-bd9f-5f18657916af
+    name: Investigate Azure Kubernetes Cluster Suspicious Operations
```

Comment:
@rosplk : I dont find this Detection search listed in this story, if we dont have a search then we should remove this story file from the PR